test/index.js

```diff
@@ -0,0 +1,9 @@
+const remoteffprobe = require('..');
+
+describe('remote-ffprobe', function () {
+  it('works', async function () {
+    this.timeout(10000);
+    const metadata = await remoteffprobe('https://s3.amazonaws.com/contentsamurai.com/users/1394472/videos/149464/video.mp4?AWSAccessKeyId=AKIAJC2SSEDWQDBZ56UA&Expires=1529485966&Signature=q4LZwArvNtL%2FsuY1rmJOi%2FpuDZ0%3D');
```

Comment:
this is going to break in a week or so... may want to sign this or better yet, use a public file for testing?

Reply:
Good point!  Will fix.